src/app/tooltip/tooltip.directive.ts

```diff
@@ -53,9 +53,7 @@ export class TooltipDirective implements OnInit, OnDestroy {
   @HostListener('touchstart', ['$event'])
   // @ts-ignore
   show(event) {
-    event.stopPropagation();
```

Comment:
@immortalityidle I removed this because it interferes with the ability to scroll the widgets. Is there any reason this is needed?

Reply:
I think I put that on so that if you had multiple tooltipped elements nested in each other you should only get the popup for first one, not the whole stack.

Reply:
That helps a lot. Used a different method that won't supress the event for the carousel.